integration-tests/spring-boot-properties/README.md

```diff
@@ -0,0 +1,12 @@
+### Current differences with Spring Boot properties.
+
+1. Default values assigned with `=` operator are not supported.
```

Comment:
Are you sure this is true?

Reply:
My bad, this readme wasn't supposed to be in this PR and is not fully correct. I'll remove it. `=` operator works for Spring Boot properties